### PR TITLE
KFSPTS-21138 backport FINP-7366

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml [hunk 1]
```diff
@@ -90,5 +90,18 @@
           p:businessObjectService-ref="businessObjectService"
           p:accountingPeriodService-ref="accountingPeriodService"
           p:dateTimeService-ref="dateTimeService" />
+          
+    <bean id="contractsGrantsBillingAwardVerificationService"
+          parent="contractsGrantsBillingAwardVerificationService-parentBean"/>
+    <bean id="contractsGrantsBillingAwardVerificationService-parentBean" abstract="true"
```
Comment (cah292) [inline src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml:96]: Was there a particular reason why you had to override the service bean's "parentBean" variant? Could this be simplified to override just the service bean?

**Review by cah292 — COMMENTED** [review]
I had a quick question about the service bean setup, but otherwise the changes look good.

File: src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml (hunk 1 above)
Comment (jhulslander) [reply src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml]: yes, the simplified spring works, thanks for noticing

**Comment by jhulslander** [comment]
This is ready for review again

**Review by cah292 — APPROVED** [review]
Changes look good, unit tests pass, and my concerns have been addressed. Merging.

**Pull request merged**